CVLByExample/Summarization/UserDefinedReturnType/runLibraryVSContractSummary.conf

```diff
@@ -5,5 +5,6 @@
     "optimistic_loop": true,
     "verify": "Caller:LibraryVSContractSummary.spec",
     "rule_sanity": "basic",
+    "solc": "solc8.10",
```

Comment:
don't have solc. the default is solc and most users use solc-select that sets the version

Reply:
Yes but then the conf won't work for users that don't, and the CI could fail because of these as well. 
Maybe I can add a note about this in the readme? Something along the lines of: "This conf uses solc8.10 as required to compile the contracts, if you wish to run it with a different version feel free to change the conf."